resources/views/docs/_websocket_events.md

```diff
@@ -0,0 +1,84 @@
+
+# Websocket Events
+
+Websocket events generally have the following standard format:
+
+```json
+{
+  "data": {},
+  "event": "some.event"
+}
+```
+
+Field | Type    | Description
+----- |-------- | -------------
+event | string  | Name of the event
+data  | object? | Event payload
+
+## `logout` event
+
+User session using same authentication key has been logged out (not yet implemented for OAuth authentication).
+Server will disconnect session after sending this event so don't try to reconnect.
+
+## `new` event
+
+Sent when a new notification is received.
+
+### Payload Format
+
+See [Notification](#notification) object for notification types.
+
+## `read` event
+
+Sent when a notification has been read.
+
+<aside class="notice">
+  This event does not use the `data` property for payload.
+</aside>
+
+TODO: `ids` should be moved to `data` to match other events.
+
+Field | Type     | Description
+----- | -------- | ----------------------------------
+event | string   | `read`
+ids   | number[] | id of Notifications which are read
+
+## chat.channel.join
```

Comment:
`logout` and `new` have their titles intentionally set like that so the generated ids are like `logout-event` and not just just `logout` since there doesn't seem to be an option to customize the id

Reply:
ಠ_ಠ

at least it shouldn't be part `<code>...</code>` and part unstyled